### ISSUE TITLE
Artifacts are failing to upload

### ISSUE THREAD

**Issue by rjollos** [rjollos/e4990a#1]
It may be similar to [Too many uploads resulting in 429 responses](https://github.com/actions/upload-artifact#too-many-uploads-resulting-in-429-responses).

```
Run actions/upload-artifact@v2
  with:
    name: e4990a
    path: dist/*
    if-no-files-found: warn
  env:
    pythonLocation: C:\hostedtoolcache\windows\Python\3.9.9\x64
With the provided path, there will be 1299 files uploaded
Starting artifact upload
For more detailed logs during the artifact upload process, enable step-debugging: https://docs.github.com/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging#enabling-step-debug-logging
Artifact name is valid!
Container for artifact "e4990a" successfully created. Starting upload of file(s)
Uploaded D:\a\e4990a\e4990a\dist\win\e4990a\libopenblas.XWYDX2IKJW2NMTWSFYNGFUWKQU3LYTCZ.gfortran-win_amd64.dll (90.2%) bytes 0:8388607
Total file count: 1299 ---- Processed file #90 (6.9%)
Uploaded D:\a\e4990a\e4990a\dist\win\e4990a\libopenblas.XWYDX2IKJW2NMTWSFYNGFUWKQU3LYTCZ.gfortran-win_amd64.dll (100.0%) bytes 8388608:9297022
Total file count: 1299 ---- Processed file #248 (19.0%)
Error: Unexpected response. Unable to upload chunk to https://pipelines.actions.githubusercontent.com/pTKxe47Sv7MqpfpakigL8ST6bLlPkDz5fBXPrREYyPAzEdXe14/_apis/resources/Containers/21488663?itemPath=e4990a%5Cwin%5Ce4990a%5Cpyinstaller-4.7.dist-info%5CREQUESTED
##### Begin Diagnostic HTTP information #####
Status Code: 400
Status Message: Bad Request
Header Information: {
  "cache-control": "no-store,no-cache",
  "pragma": "no-cache",
  "transfer-encoding": "chunked",
  "content-type": "application/json; charset=utf-8",
  "strict-transport-security": "max-age=2592000",
  "x-tfs-processid": "9f3f0a80-9c4e-452e-927d-dac78a5e8339",
  "activityid": "9055eea3-6320-4e6d-b5f9-dd659cbd074d",
  "x-tfs-session": "9055eea3-6320-4e6d-b5f9-dd659cbd074d",
  "x-vss-e2eid": "9055eea3-6320-4e6d-b5f9-dd659cbd074d",
  "x-vss-senderdeploymentid": "a07ab14e-025a-39c3-8d53-788cd7ce207f",
  "x-frame-options": "SAMEORIGIN",
  "x-cache": "CONFIG_NOCACHE",
  "x-msedge-ref": "Ref A: A089E2E7A06B4191A1AB4B73EB20D863 Ref B: DM2EDGE0621 Ref C: 2021-12-08T00:14:34Z",
  "date": "Wed, 08 Dec 2021 00:14:34 GMT"
}
###### End Diagnostic HTTP information ######
Warning: Aborting upload for D:\a\e4990a\e4990a\dist\win\e4990a\pyinstaller-4.7.dist-info\REQUESTED due to failure
Error: aborting artifact upload
Total size of all the files uploaded is 37534204 bytes
File upload process has finished. Finalizing the artifact upload
Upload finished. There were 978 items that failed to upload

The raw size of all the files that were specified for upload is 91907447 bytes
The size of all the files that were uploaded is 37534204 bytes. This takes into account any gzip compression used to reduce the upload size, time and storage

Note: The size of downloaded zips can differ significantly from the reported size. For more information see: https://github.com/actions/upload-artifact#zipped-artifact-downloads 

Error: An error was encountered when uploading e4990a. There were 978 items that failed to upload.
```

**Comment by rjollos** [comment]
The issue appears due to empty files, as described in [upload-artifact #150](https://github.com/actions/upload-artifact/issues/150).

```
Error: Unexpected response. Unable to upload chunk to https://pipelines.actions.githubusercontent.com/pTKxe47Sv7MqpfpakigL8ST6bLlPkDz5fBXPrREYyPAzEdXe14/_apis/resources/Containers/21488663?itemPath=e4990a%5Cwin%5Ce4990a%5Cpyinstaller-4.7.dist-info%5CREQUESTED
```

`itemPath` decodes `itemPath=e4990a%5Cwin%5Ce4990a%5Cpyinstaller-4.7.dist-info%5CREQUESTED` to `itemPath=e4990a\win\e4990a\pyinstaller-4.7.dist-info\REQUESTED`.

On macOS the `REQUESTED` files are empty, and presumably also on Windows.
```
$ find . -name REQUESTED
./dist/mac/e4990a/wheel-0.37.0.dist-info/REQUESTED
./dist/mac/e4990a/setuptools-59.5.0.dist-info/REQUESTED
```

**Comment by rjollos** [comment]
The issue seems to be resolved now as re-running [#98](https://github.com/rjollos/e4990a/actions/runs/1551824364) succeeded.
